### DISCUSSION TITLE
test discussion

### DISCUSSION THREAD

**Discussion by Dave-12138** [Dave-12138/miyoushe-emoticon-in-discussion#1] [其他]
emm

**Comment by Dave-12138** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/ef939dd02fa4e48c6de1ca2ad661ac88_5558307353343114792.png" alt="萨姆 出击" width="75" >

**Comment by lpifx** [comment]
<img src="https://bbs-static.miyoushe.com/static/2025/07/02/4eaee0147a7021dbaba94663be2ecfba_5233714634604237118.png" alt="白厄 掉线" width="75" >

**Reply by Dave-12138** [reply]
<img src="https://bbs-static.miyoushe.com/static/2025/01/15/88b6990f6b2b49bbdc52e1c4bb1728a1_3653258358045665724.png" alt="大地兽 探头" width="75" >
